jerry-core/parser/js/js-parser-expr.c

```diff
@@ -3398,19 +3398,21 @@ parser_process_group_expression (parser_context_t *context_p, /**< context */
   parser_stack_pop_uint8 (context_p);
   lexer_next_token (context_p);
 
-  if (context_p->token.type == LEXER_ASSIGN)
+#if ENABLED (JERRY_ES2015)
+  if (context_p->token.type == LEXER_ASSIGN
+      && parser_is_assignment_expr (context_p))
```

Comment:
What about limiting this to identifiers?

Reply:
Good idea.